### DISCUSSION TITLE
An idea to improve the ''full'' setup instructions.

### DISCUSSION THREAD

**Discussion by Art3mis-Ghub** [vladmandic/sdnext#2312] [Ideas]
Hi,

I think it would be a good idea to add the step where you explain how to include ''--debug'' option to the webui.bat for the initial setup, for new users that isn't immediatelly obvious how to do, and while it's not really required, since it's advisable it would be a good idea to ensure that new users understand how to do this when they install for the first time.

Just a thought :)

**Comment by vladmandic** [comment]
that's what https://github.com/vladmandic/automatic/discussions/1011 is for
